vllm/entrypoints/llm.py

```diff
@@ -1045,37 +1037,92 @@ def ensure_str(prompt: SingletonPrompt):
         if len(text_1) == 1:
             text_1 = text_1 * len(text_2)
 
-        input_pairs = [(t1, t2) for t1, t2 in zip(text_1, text_2)]
-        pooling_params = PoolingParams()
+        if self.llm_engine.model_config.task != "score":
```

Comment:
```suggestion
if self.llm_engine.model_config.task not in ["embed", "score"]:
```

Reply:
It'd be nice to move this up to the top of the method as well, before other work is done